text/3392-leadership-council.md

```diff
@@ -226,17 +226,17 @@ If this constraint does not hold, whether by a representative changing affiliati
 
 儘管有些團隊目前可能沒有足夠的候選人符合這些標準，理事會應積極培養這些技能之人才，因為這些技能不僅對理事會成員資格有幫助，也對這個專案有益。
 
-## 與 Core 團隊之關係
+## 與核心團隊之關係
 
-The Leadership Council serves as the successor to the core team in all capacities. This RFC was developed with the participation and experience of the core team members, and the Council should continue seeking such input and institutional memory when possible, especially while ramping up.
+領導理事會繼承核心團隊所有職務。這份 RFC 是在核心團隊成員的參與和經驗下開發的，在可能的情況下理事會應該繼續尋求類似的意見與機構記憶，特別是在擴張（組織）時。
 
-External entities or processes may have references to "the Rust core team" in various capacities. The Council doesn't use the term "core team", but the Council will serve in that capacity for the purposes of any such external references.
+外部實體或流程可能以各種身份提及「*Rust 核心團隊*」。理事會並不使用「核心團隊」這個名稱，但理事會將在這些外部參考需要時作為這樣的角色存在。
 
 ## 與 Rust 基金會之關係
 
-The Council is responsible for establishing the process for selecting Project directors. The Project directors are the mechanism by which the Rust Project's interests are reflected on the Rust Foundation board.
+理事會負責建立選擇專案管理者的流程。專案管理者在基金會理事會上反映 Rust 專案利益。
 
-The Council delegates a purview to the Project directors to represent the Project's interests on the Foundation Board and to make certain decisions on Foundation-related matters. The exact boundaries of that purview are out of scope for this RFC.
+理事會授權專案管理者在基金會理事會上代表 Rust 專案的利益，並且在基金會相關事項上做出某些決策。授權的確切界線則超出此 RFC 的討論範圍。
```

Comment:
* Project director is a proper noun, a role in Rust Foundation. We'd better keeping it here.
* Foundation Board as well, though I am not sure which one we want (see https://github.com/rust-tw/rfcs-tw/commit/ce1c762a187abd3bd7d1e58669f8e0a740b45b3e).

```suggestion
理事會授權專案管理者（Project director）在基金會理事會上代表 Rust 專案的利益，並且在基金會相關事項上做出決策。授權的確切界線則超出此 RFC 的討論範圍。
```